packages/cli/constants/constants.ts

```diff
@@ -9,6 +9,11 @@ export const CREATE_MESSAGES = {
   extensions: "What extensions do you want to add to your contract?",
 } as const;
 
+export const GENERATE_MESSAGES = {
+  deployerAddress: "What's the deployer address of your smart contracts?",
```

Comment:
would say something simpler like: 

`Deployer wallet address: ...`

maybe we need to explain that this will pull your dashboard contracts somewhere